src/main/Config.cpp

```diff
@@ -128,7 +128,7 @@ Config::Config() : NODE_SEED(SecretKey::random())
     MAXIMUM_LEDGER_CLOSETIME_DRIFT = 50;
 
     OVERLAY_PROTOCOL_MIN_VERSION = 19;
-    OVERLAY_PROTOCOL_VERSION = 21;
```

Comment:
careful when rebasing here (should be 23)

Reply:
Done, thanks for the reminder.